### DISCUSSION TITLE
in-network-rates "Provider Reference Object" Section - Bad Links

### DISCUSSION THREAD

**Discussion by alloyg** [CMSgov/price-transparency-guide#417] [Q&A] [answered]
I noticed that almost all of the hyperlinks in the "Provider Reference Object" section end up on "Page not found" pages.  Any chance these will be updated?  Thanks.

[https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#provider-reference-object](url)

**Comment by shaselton-usds** [comment, marked as answer]
Everything should be updated to the specific commit to avoid 404s when HEAD files are changed.
